src/Storefront/Test/Controller/CartLineItemControllerTest.php

```diff
@@ -40,6 +48,41 @@ public function testErrorBehaviourInFlashMessages(): void
         static::assertTrue($response->isRedirect(), $response->getContent());
     }
 
+    /**
+     * @dataProvider productNumbers
+     */
+    public function testAddProductByNumber(string $productId, string $productNumber): void
+    {
+        $this->getFlashBag()->clear();
```

Comment:
is this necessary?

Reply:
oh I see

Reply:
Nope, left over from testing around. good catch :)

Reply:
isn't it necessary for clearing the flashbag? because you test on it. I think not clearing it, would pollute the test, won't it?